### PR TITLE
added no data message for the shop and machine

### PR DESCRIPTION
**Pull request by manojflyer**
- created no data message for the shop and machine

### CODE REVIEW

File: src/pages/plants/index.tsx [hunk 1]
```diff
@@ -96,11 +100,14 @@ function Plant() {
                 </div>
               )}
               {show === 'plant' &&
-                plants?.data?.map((plant: any) => (
-                  <div key={plant?.plantId} className={`${plant_card}`}>
-                    <PlantCard name={plant?.plantName} image={plant?.image} onClick={() => plantBtn(plant)} />
-                  </div>
-                ))}
+                plants?.data?.map((plant: any) => {
+                  console.log('==plant', plant);
```
Comment (JayshreeFlyers) [inline src/pages/plants/index.tsx:104]: remove console

File: src/components/reusable/loading/index.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const Loading = () => {
   return (
     <>
-      <span className="absolute w-[30px] h-[30px] bg-white rotate-45 top-[-15px] left-[95px]"></span>
+      {/* <span className="absolute w-[30px] h-[30px] bg-white rotate-45 top-[-15px] left-[95px]"></span> */}
```
Comment (chandran-kannan) [inline src/components/reusable/loading/index.tsx:4]: remove the commented line if its not needed for future reference

File: src/pages/plants/PlantCard.tsx [hunk 3]
```diff
@@ -4,7 +4,7 @@ const PlantCard = ({ image, name, onClick }: CardType) => {
   return (
     <div onClick={onClick}>
       <img className="w-[200px] h-[180px]" src={image} alt={name} />
-      <p className="text-center pt-5 text-lg">{name}</p>
+      <p className="w-full text-center pt-1 text-lg inline-block">{name}</p>
```
Comment (chandran-kannan) [inline src/pages/plants/PlantCard.tsx:7]: why did you add w-full and inline block?
because <p> tag is block element, so if you want inline-block then use <span> tag

**Review by chandran-kannan — CHANGES_REQUESTED** [review]

File: src/pages/plants/index.tsx [hunk 4]
```diff
@@ -109,42 +116,66 @@ function Plant() {
...
+                            ))}
                           </div>
-                        ))}
-                      </div>
+                        </>
+                      ) : (
+                        <>
+                          <span className="absolute w-[30px] h-[30px] bg-white rotate-45 top-[-15px] left-[335px]"></span>
+                          <p className="text-[#492CE1] font-GothamMedium text-lg font-medium">
+                            No Shops are created!!!
```
Comment (chandran-kannan) [inline src/pages/plants/index.tsx:137]: can you verify this text with design team `No Shops are created!!!`

**Review by chandran-kannan — CHANGES_REQUESTED** [review]

**Review by chandran-kannan — APPROVED** [review]
